tools/llm_bench/llm_bench_utils/config_class.py

```diff
@@ -102,7 +102,9 @@
         "olmo",
         "phi3",
         "starcoder",
-        "instruct-gpt"
+        "instruct-gpt",
+        "granite",
+        "granitemoe",
```

Comment:
BTW, why do we need to maintain all these lists of models?

Reply:
properly assign task for them, but I agree it will much convenient if we do not need it. The only problem is lack of ability to get model task from local directory on optimum side (I have some ideas how to improve that, but need some time to finish my  POC)